extra/make_runnerrpms.sh

```diff
@@ -1,11 +1,11 @@
 #!/bin/sh
 
-if [ $# -gt 0 -o "$1" == "--help" -o "$1" == "help" ]; then
+if [ "$1" == "--help" -o "$1" == "help" ]; then
 	echo ""
 	echo "  USAGE:"
 	echo ""
 	echo "  # cd tcmu-runner/extra/"
-	echo "  # ./make_runnerrpms.sh"
+	echo "  # ./make_runnerrpms.sh [--without rbd] [--without gluster] [--without qcow] [--without zbc]"
```

Comment:
Will it be better to using glfs instead of gluster here ? Which will be keep the same with the handler name.

Reply:
I thought of it, but somehow gluster sounded better. I can make that change. No issues

Reply:
And I am also okay with this.